apps/VC3D/OpsList.cpp

```diff
@@ -11,29 +11,31 @@ OpsList::OpsList(QWidget* parent) : QWidget(parent), ui(new Ui::OpsList)
 {
     ui->setupUi(this);
 
-    _tree = this->findChild<QTreeWidget*>("treeWidget");
-    _add_sel = this->findChild<QComboBox*>("comboOp");
+    _tree = ui->treeWidget;
+    _add_sel = ui->comboOp;
     connect(_tree, &QTreeWidget::currentItemChanged, this, &OpsList::onSelChanged);
-    connect(this->findChild<QPushButton*>("pushAppendOp"), &QPushButton::clicked, this, &OpsList::onAppendOpClicked);
+    connect(ui->pushAppendOp, &QPushButton::clicked, this, &OpsList::onAppendOpClicked);
```

Comment:
Not related to the issues, but I think in general we should prevent those unnecesarry `findChild` calls.